build-ci/third_party.rst

```diff
@@ -177,6 +184,36 @@ gzipped tarball in :file:`upstream/`.
    EUPS expects the patches to be formatted according to the output of
    :command:`git diff`, not the output of :command:`diff`.
 
+.. _other-files:
```

Comment:
Could you namespace this label to the page (these labels are global across the entire doc)? Maybe

```
.. _build-third-party-other-files:
```